### DISCUSSION TITLE
Starting a wiki for pypsa-earth

### DISCUSSION THREAD

**Discussion by Tomkourou** [pypsa-meets-earth/pypsa-earth#745] [Ideas]
It occurred to me that we don't really have a place yet where validation efforts are documented. 

A wiki might be a nice place to add pages about countries/regions that are being validated, who is working on them, what the data workflows are, what the pitfalls to lookout for are etc. 

I also see that it's not yet fully clear how one might start validating a new region. For Africa it seems the workflow is well set up. For other regions like SEA tinkering in the config is required to get it working. We could add more information about config files in the wiki to help people get started quickly. 

@ekatef @pz-max @davide-f what do you think?

**Comment by ekatef** [comment]
Absolutely agree that we may need to rethink a way how validation is documented. We have tried to use issues to track a validation status, but it doesn't seem to be a really convenient way. There are some materials related to validation methodology and results, but they are quite scattered currently. 

Probably, wiki could work very well to help navigate through existing content and facilitate creation new one. Although, personally I don't have much experience with wikis.
@Tomkourou do you have an idea on how much efforts does wiki maintenance require?

**Comment by Tomkourou** [comment]
I don't think it would be too time consuming. They are just markdown files at the end of the day. We can update them as of and when people have new information/data on the validated country. The main thing would be to create a solid template that we can use to populate a country or region validation page with.
